BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/MainActivity.java

```diff
@@ -42,13 +46,20 @@ protected void onCreate(Bundle savedInstanceState) {
                     Toast.makeText(getApplicationContext(),
                             "Created session!",
                             Toast.LENGTH_LONG).show();
-
+                    /*
```

Comment:
Please don't commit commented code. Remove completely. We always have this code in our history if needed.